src/helpers/displayErrorMessagesAddUser.js

```diff
@@ -0,0 +1,23 @@
+import { errors } from '../helpers/errorMessages'
+import { errorHandle } from './errorMessageHandle'
+
+export const displayErrorMessagesAddUSer = (allFieldsFilledIn, emailValidation, phoneValidation, doubleUserValidation, doubleUserValidationInput, dispatchGetFormError, formData) => {
+
+  const emptyField = allFieldsFilledIn === false ? errors.errorNotAllFieldsAreFilledIn : null
+  errorHandle(emptyField, dispatchGetFormError)
+
+  const emailIsValid = formData.name.length > 0 && formData.lastName.length > 0 && formData.phone.length > 0 && emailValidation === null ? errors.errorEmailIsNotValid : null
+  errorHandle(emailIsValid, dispatchGetFormError)
+
+  const phoneIsValid = formData.name.length > 0 && formData.lastName.length > 0 && emailValidation !== null && phoneValidation === null ? errors.errorPhoneIsNotValid : null
+  errorHandle(phoneIsValid, dispatchGetFormError)
+
+  const allDataIsValid = formData.name.length > 0 && formData.lastName.length > 0 && emailValidation !== null && phoneValidation !== null ? errors.userAdded : null
+  errorHandle(allDataIsValid, dispatchGetFormError)
+
+  const doubleUser = doubleUserValidation.length > 0 ? errors.userExists : null
+  errorHandle(doubleUser, dispatchGetFormError)
+
+  const doubleUserInput = doubleUserValidationInput.length > 0 ? errors.userExists : null
+  errorHandle(doubleUserInput, dispatchGetFormError)
+}
```

Comment:
Ten kod można uprościć / usprawnić na wiele różnych sposobów. Przede wszystkim po to, żeby łatwiej się go czytało innym programistom. A to musisz mieć na względzie, gdy tworzysz kod. Ty nie będziesz się nim zajmował przez wieczność, w końcu trafi do innego człowieka. Pisz kod w taki sposób, żeby inni nie mieli problemów z jego czytaniem i rozumieniem. Czy chciałbyś otrzyma w spadku coś takiego od innego programisty? Przegadajmy jutro ten plik i spróbujmy go razem zrefactorować :)